doc/contributing/docs/markup/code.rst

```diff
@@ -124,6 +124,9 @@ And you will get this:
 
     ..  include:: _includes/samp.rst
 
+As you can see, with ``:extsamp:`` you don't need to mix ``:samp:`` with ``:code:``
```

Comment:
You really don't need it with `:samp:` either, only when there are `{` or `}` in the code. It seems to me that the purpose of `extsamp` is to enable italic and bold in the code samples.

Reply:
Please add an example of
```
:extsamp:`bla bla **variable** foo bar`
```